modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -5,40 +5,97 @@
 // * installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc
 
 [id="installation-vsphere-installer-infra-requirements_{context}"]
-= vCenter requirements
+= Requirements for vCenter cluster install
 
-Before you install an {product-title} cluster on your vCenter that uses infrastructure that the installer provisions, you must prepare your environment.
+Before you install an {product-title} cluster on your vCenter that uses installer-provisioned infrastructure, you must prepare your environment. Logging in to a user account or group that has administrative privileges is the simplest way to access all of the necessary permissions in vCenter.
+
+By configuring roles and permissions, vCenter authentication determines how a group or user can access vSphere components and objects, both globally and locally.
 
 [discrete]
 [id="installation-vsphere-installer-infra-requirements-account_{context}"]
-== Required vCenter account privileges
-
-To install an {product-title} cluster in a vCenter, the installation program requires access to an account with privileges to read and create the required resources. Using an account that has administrative privileges is the simplest way to access all of the necessary permissions.
-
-A user requires the following privileges to install an {product-title} cluster:
-
-* Datastore
-** *Allocate space*
-** *Browse datastore*
-** *Low level file operations*
-** *Remove file*
-* Folder
-** *Create folder*
-** *Delete folder*
-* vSphere Tagging
-** All privileges
-* Network
-** *Assign network*
-* Resource
-** *Assign virtual machine to resource pool*
-* Profile-driven storage
-** All privileges
-* vApp
-** All privileges
-* Virtual machine
-** All privileges
-
-For more information about creating an account with only the required privileges, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-5372F580-5C23-4E9C-8A4E-EF1B4DD9033E.html[vSphere Permissions and User Management Tasks] in the vSphere documentation.
+== Applying vCenter account privileges
+
+To install an {product-title} cluster in a vCenter, the installation program requires access to an account with privileges to read and create the required resources.
+
+[IMPORTANT]
+====
+It is recommended that you apply role access privileges globally to ensure that the required permissions are met. If these permissions are not applied globally, you must determine which entities to grant the role permissions to. At minimum, the user or group must have `read-only` access for both the vCenter and the vSphere entities.
+====
+
+.Procedure
+To configure your vCenter environment so that a user in {product-title} has the necessary privileges to provision storage for the installer:
+
+. Create a custom role in vCenter if you have not already done so. By default, `read-only` permission is granted when you create a custom role.
```

Comment:
>  By default, `read-only` permission is granted when you create a custom role.

I don't think this is true. See: https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.security.doc/GUID-93B962A7-93FA-4E96-B68F-AE66D3D6C663.html

Read only is a System Role and that document states: 
> Roles that you create do not inherit privileges from any of the system roles.

Instead of creating a custom role for storage, we should just tell them to apply permissions for the Read Only system role to the node VM and all of its parent entities (|vCenter, Datacenter, Datastore Cluster, Datastore Storage Folder
).

Reply:
> Instead of creating a custom role for storage, we should just tell them to apply permissions for the Read Only system role to the node VM and all of its parent entities 

If I understand this statement correctly - `ReadOnly` role is not enough for storage provisioning. We actually *do* need custom permissions mentioned in this doc, *in addition* to `ReadOnly` role.

Reply:
Seems like their docs are in contradiction. See: https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.security.doc/GUID-18071E9A-EED1-4968-8D51-E0B4F526FDA3.html

> Note:
> When you add a custom role and do not assign any privileges to it, the role is created as a Read Only role with three system-defined privileges: System.Anonymous, System.View, and System.Read.

Reply:
@patrickdillon While you say, "instead of creating a custom role for storage", @gnufied contends in Slack that "we need two roles to apply at vcenter level and propagate to all the children". So I'm not sure whether we should have additional steps (maybe between the two tables), something like:

> 
> `...`
> . Create a custom role in vCenter with read-only privileges.
> 
> . Apply the new custom role to the OpenShift Container Platform user at the vCenter level.
> 
> . Propagate the new custom role set to all the child entities of the node VM object at the vCenter level, as shown in the following table:
> 
> **Table 2. Required vSphere levels**

> |===
> |Level
> |Privileges
> 
> |Datastore
> |* *Allocate space*
> * *Browse datastore*
> * *Low level file operations*
> * *Remove file*

> `...`

I'm going to go push a new commit that updates with these additions, though I'm not certain I have the wording or ordering quite right.